src/ui_helpers.c

```diff
@@ -10,14 +10,15 @@
 // We use this trick since only one of the two may happen at any time.
 ui_display_state_t displayState;
 
+// make sure the SDK didn't switch to multi-byte character sets.
+STATIC_ASSERT(SIZEOF(uint8_t) == SIZEOF(char), "bad char size");
```

Comment:
Remove, `SIZEOF(uint8_t) == SIZEOF(char)` is a C constraint

Reply:
Done